README.md

```diff
@@ -1 +1,107 @@
-# metrics-webapp
\ No newline at end of file
+# Hogwarts Mystery 🧙
+
+
+## Description
+**A mobile web application that allows the user to create a list of metrics by utilizing React and Redux. 
+**The webapp consists of two pages: the first page will display a list of items that can be filtered using the name, serving as the homepage. The second page will be for displaying the details of a particular item.** 
+# 🛠 Built With
+### Tech Stack:
+- React
+
+### Client:
+- React
+
+# 🚀 Live Demo <a name="live-demo"></a>
+
+
+
+<img src="pic.png" alt="screen shot">
```

Comment:
- [x] While you make a great effort I see that you forgot the link to the live version of the project to let the user be able to see the demo of the project, but here I can not 😊 see the link for the demo, so please try to add the live demo version link using Render, Netlify or GitHub Pages to let the user see the demo of the project  😊⤵️

| Project requirements |
| ---|   
| ![image](https://user-images.githubusercontent.com/83360636/233748929-6ccd5eac-2750-4e54-8533-029c5a6f3aa6.png)  |